plugin/cache/README.md

```diff
@@ -50,6 +51,10 @@ cache [TTL] [ZONES...] {
   **DURATION** defaults to 1m. Prefetching will happen when the TTL drops below **PERCENTAGE**,
   which defaults to `10%`, or latest 1 second before TTL expiration. Values should be in the range `[10%, 90%]`.
   Note the percent sign is mandatory. **PERCENTAGE** is treated as an `int`.
+* `serve_expired`, when set to yes, it will serve expired responses from cache when there are no healthy
+  upstream backends. The responses have a TTL of 0.  **DURATION** is how far back to consider stale
```

Comment:
"... when there are no healthy upstream backends."

IIUC, we are not literally watching the health of the upstream backends here.  So this may be misleading.  One might infer that if an expired entry is served to the client, that must mean that the backend/upstream is unhealthy/non-responsive - but that is not always the case.  The actual behavior may be harder to succinctly describe, but IMO we should try...

When serve_expired is set, CoreDNS always will serve an expired cache to a client if an expired cache entry exists.  When this is happens, CoreDNS will attempt to refresh the cache entry after sending the expired cache entry to the client.

Reply:
You are correct.
I've updated  the text with your more accurate description. Let me know if there's anything that looks off.

Thanks

-Gonzalo